File: 2019-2/SWPP/Lab_01/prime5to50.js
function wannaGoHome(){
    var str = ""
    for(var n = 5; n <= 50; n++){
        var isPrime = true;
        for(var probe = 2; probe < n; probe++)
            if(n % probe == 0)
                isPrime = false;

        if(isPrime)
            str += n + " is a prime number</br>"
    }

    return str
}

File: 2019-2/SWPP/Lab_01/prime5to50.py
def wannaGoHome():
    for n in range(5, 51):
        isPrime = True
        for probe in range(2, n):
            if n % probe == 0:
                isPrime = False

        if isPrime:
            print(str(n) + " is a prime number")



wannaGoHome()
